docs/platforms/javascript/common/feature-flags/index.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Set Up Feature Flags
+sidebar_order: 7000
+notSupported:
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.bun
+  - javascript.capacitor
+  - javascript.cloudflare
+  - javascript.connect
+  - javascript.cordova
+  - javascript.deno
+  - javascript.electron
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.nodejs
+  - javascript.wasm
+description: With Feature Flags, Sentry tracks flag evaluations in your application and reports their state on error. Sentry will also record an audit log of feature flag changes and report any suspicious changes that may have triggered an error.
+---
+
+<PlatformContent includePath="feature-flags/prerelease-alert" />
+
+## Prerequisite
```

Comment:
```suggestion
## Prerequisites
```

Reply:
Re-adding the "s".  You can Jasmin can argue over this before I change it again lol.

Reply:
add the `s` imo

Also, the other page has an `s` already.